ppb/engine.py

```diff
@@ -8,6 +8,19 @@
 class GameEngine(Engine):
 
     def __init__(self, first_scene: Type, **kwargs):
+        """
+        Game Engine
```

Comment:
This isn't documentation, and echoes the name of the class.

We can change this to "Create GameEngine" or ignore it since __init__'s usage is well known.